velox/functions/lib/string/StringCore.h

```diff
@@ -415,14 +417,24 @@ getByteRange(const char* str, size_t startCharPosition, size_t length) {
 
     // Find startByteIndex
     for (auto i = 0; i < startCharPosition - 1; i++) {
```

Comment:
what happens if the startCharPosition is say 10 and the invalid utf8 is at 6, and it returns a pair of <5,5>, what would be the output of this in rpad, like does it add padding at the wrong index?

Can you mention what the expected behavior is in presto? I could not find anything explicit on https://prestodb.io/docs/current/functions/string.html

Also, wondering if it makes sense to add this behavior to the function docs. So that we know what to expect.

Reply:
I validated this behaviour for both rpad/lpad in presto and velox and both give similar results. 
I am adding additional test to pad for this purpose though.